standard/expressions.md

```diff
@@ -182,11 +182,11 @@ All unary and binary operators have predefined implementations. In addition, use
 
 The ***overloadable unary operators*** are:
 
-```csharp
-+  -  !  ~  ++  --  true  false
-```
+`+  -  !` (logical negation only) `~  ++  --  true  false`
 
 > *Note*: Although `true` and `false` are not used explicitly in expressions (and therefore are not included in the precedence table in [§12.4.2](expressions.md#1242-operator-precedence-and-associativity)), they are considered operators because they are invoked in several expression contexts: Boolean expressions ([§12.24](expressions.md#1224-boolean-expressions)) and expressions involving the conditional ([§12.18](expressions.md#1218-conditional-operator)) and conditional logical operators ([§12.14](expressions.md#1214-conditional-logical-operators)). *end note*
+>
```

Comment:
This will produce the format we want in the final Word output without the markdown lint warnings:

```suggestion
<!-- markdownlint-disable MD028 -->

<!-- markdownlint-enable MD028 -->
```